src/plugins/presentation_util/public/components/field_picker/field_picker.tsx

```diff
@@ -40,7 +40,7 @@ export const FieldPicker = ({
         dataView.fields
           .filter(
             (f) =>
-              f.name.includes(nameFilter) &&
+              f.name.toLowerCase().includes(nameFilter.toLowerCase()) &&
```

Comment:
Tested this - works exactly as expected! Searching for `CURR` shows me `currency` as a suggested field.